charts/medusa-operator/templates/deployment.yaml

```diff
@@ -1,3 +1,5 @@
+{{- $defaultPodSecurityCtx := "{}" }}
```

Comment:
`podSecurityContext` is already defined in medusa-operator/values.yaml. Why redefine it here?